src/test/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/service/RiskServiceTest.kt

```diff
@@ -254,7 +254,7 @@ internal class RiskServiceTest : ServiceTestBase() {
 
       // then
       assertThat(response?.lastUpdatedDate).isEqualTo("2022-08-26T15:00:08.000Z")
-      assertThat(response?.offenceDataFromLatestCompleteAssessment).isEqualTo(true)
```

Comment:
Just wondering if the test description can be improved for these 2 that you've changed here? The business rules here are quite complicated and it's not clear to me why this needs to be changed from true to false following the change you made in RiskService. I can't see a test to cover when the offenceDataFromLatestCompleteAssessment field should be true, is this not a valid scenario to test for?

Reply:
I opted to rename the variable tested in the assert to give more clarity.